### PR TITLE
Update color-blue to #006fbb

### PR DESCRIPTION
**Pull request by alex-page**
This changes `color-blue` to `#006fbb` from `#007ace` so that our link color passes contrast 4.5 on our currently darkest background colour `#fbeae5`.

This will allow these issues to be closed: 
- https://github.com/Shopify/polaris-react/issues/1619
- https://github.com/Shopify/polaris-ux/issues/148

* [x] Style guide
* [x] Sketch UI Kit cc @sarahill 
* [x] Sketch palette 
* [x] Icons Sketch Kit
* [x] Figma styles @HYPD

### CODE REVIEW

**Comment by HYPD** [comment]
@alex-page has this been updated everywhere?
* Style guide
* UI Kit cc @sarahill 
* Figma (I can do this)
* Anywhere else we do this

**Comment by kaelig** [comment]
cc @jessebc, as creative direction owns the palette.

**Comment by alex-page** [comment]
For context this should not invoke a "visual" change. It is shifting the lightness of the link blue by 3% so that it meets accessibility across all of our background colours. It is currently failing on white, light grey and our darkest background colour the light red.

**Comment by sarahill** [comment]
I think the style guide will get the color updates once this PR is merged. Correct me if I'm wrong @HYPD @alex-page

**Comment by kaelig** [comment]
To be fully thorough we would look at all the styleguide images and modify those that include this blue to relfect the change, but that's probably a lot of work for very little return. Thoughts?

When it comes to the color palette shown in the style guide, a dependency update is all that's needed.

**Comment by sarahill** [comment]
I feel like updating all the images would be little impact. Let's take the time to update it in the kits and palettes where the colors will be pulled from but images seem low for this particular update.

Open to other opinions on this though. That's just my initial thought.

**Comment by alex-page** [comment]
For this change unless the image has the hexadecimal value in it I would avoid changing it. This is not meant to be visually perceived as different from the previous colour.

**Comment by HYPD** [comment]
@alex-page Figma styles updated

**Comment by alex-page** [comment]
@kaelig not sure why that is happening. For context if run:
```shell
yarn get-tokens-from-invision-dsm
yarn dist
yarn test -u
```

Locally, I can also see the change to `colorYellowDark`. Maybe the colour was updated accidentally in invision DSM or it was updated and never added back to the tokens.

I put a comment next to the file with the updated filter, this was done in the original commit.

File: data/color-metadata.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ color-blue-darker:
 color-blue-dark:
   filter: brightness(0) saturate(100%) invert(22%) sepia(70%) saturate(1308%) hue-rotate(182deg) brightness(94%) contrast(101%)
 color-blue:
-  filter: brightness(0) saturate(100%) invert(32%) sepia(99%) saturate(1186%) hue-rotate(181deg) brightness(91%) contrast(103%)
+  filter: brightness(0) saturate(100%) invert(19%) sepia(98%) saturate(2885%) hue-rotate(190deg) brightness(99%) contrast(101%);
```
Comment (alex-page) [inline data/color-metadata.yml]: @kaelig filter added here, following docs:
> When updating a color, please also remember to update its filter metadata. Filter values can be generated using this CodePen and are stored in /data/color-metadata.yml.

**Comment by kaelig** [comment]
I looked into it, and it seems like #44 didn't update the filter for yellow-dark, hence why it appears in this PR.

**Pull request merged**